plugin/src/software/aws/toolkits/eclipse/amazonq/configuration/profiles/QDeveloperProfileUtil.java

```diff
@@ -169,22 +159,24 @@ public synchronized CompletableFuture<Void> getProfileSelectionTaskFuture() {
         return profileSelectionTask;
     }
 
-    private boolean isValidProfile(final QDeveloperProfile profile) {
-        return profile != null && StringUtils.isNotBlank(profile.getName()) && isValidAccountId(profile.getAccountId())
-                && isValidArn(profile.getArn()) && isValidRegion(profile.getRegion());
+    private boolean isValidFetchedProfile(final QDeveloperProfile profile) {
+        return profile != null;
+    }
+
+    private boolean isValidCachedProfile(final QDeveloperProfile profile) {
+        return profile != null  && isValidAccountId(profile.getAccountId()) && isValidArn(profile.getArn()) && isValidRegion(profile.getRegion());
     }
 
     private boolean isValidAccountId(final String accountId) {
-        return accountId != null && accountId.matches("^\\d{12}$");
+        return StringUtils.isNotBlank(accountId);
     }
 
     private boolean isValidArn(final String arn) {
-        return arn != null && arn.matches("^arn:aws:codewhisperer:[a-z]{2}-[a-z]+-\\d:\\d{12}:profile/[A-Z0-9]+$");
+        return StringUtils.isNotBlank(arn);
     }
 
     private boolean isValidRegion(final String region) {
-        return region != null && region
-                .matches("^[a-z]{2}-(central|north|south|east|west|northeast|southeast|northwest|southwest)-(\\d)$");
+        return StringUtils.isNotBlank(region);
     }
```

Comment:
Why drop these validations?

Reply:
IMO it's fine to not have these check, in most scenarios, users won't touch / modify the cached file (also that's what we expect). We just need to ensure serialize/de-serialize work fine then it should be good enough.